### ISSUE TITLE
reverse the arg list for cluster()

### ISSUE THREAD

**Issue by blitmap** [Flipboard/express-cluster#4]
I see that you wrote express-cluster in Coffeescript - you distribute the compiled JS.  I think calling `cluster()` would look nicer for people using Coffeescript if you reversed the arg list:

```coffeescript
cluster { count: require('os').cpus().length + 1, respawn: true }, (worker) ->
  app = express()
  app.get '/', (req, res) -> res.send "Hello from worker ##{workder.id}!"
  app.listen 0xbeef
```

What do you think? :-)

**Comment by eden** [comment]
Agreed! I created https://github.com/Flipboard/express-cluster/pull/6 for this issue.

**Comment by eden** [comment]
Fixed in 8cf4978
